src/Client.php

```diff
@@ -10,7 +10,6 @@
 use Voronkovich\SberbankAcquiring\Exception\ResponseParsingException;
 use Voronkovich\SberbankAcquiring\HttpClient\CurlClient;
 use Voronkovich\SberbankAcquiring\HttpClient\HttpClientInterface;
-use Voronkovich\SberbankAcquiring\OrderStatus;
```

Comment:
Why did you remove this line? The `OrderStatus` class is used here: https://github.com/voronkovich/sberbank-acquiring-client/blob/452244943e88284a25ee6172d05063d3732c6f9d/src/Client.php#L337

Reply:
Yes, it used here, but OrderStatus has the same namespace...   My linter always remove imports like this.
What for you  are asking to bring it back? Just for more readable code or something more?

Reply:
@floor12, you're right! I forgot that `OrdedStatus` has the same namespace. Sorry for confusion. Indeed such imports should be removed.